provisioning/plugins/corim-psa-decoder/test-harness/build-test-vectors.sh

```diff
@@ -24,13 +24,38 @@ TV_DOT_GO=${TV_DOT_GO?must be set in the environment.}
 
 printf "package main\n\n" > ${TV_DOT_GO}
 
-for t in ${COMID_TEMPLATES}
-do
-	cocli comid create -t ${t}.json
-	cocli corim create -m ${t}.cbor -t ${CORIM_TEMPLATE} -o corim${t}.cbor
+generate_cbor () {
```

Comment:
the function name is misleading: it should be called "generate_go_test_vector" or similar.

also please add a one-liner describing the purpose of the function and each of the parameters too, e.g.:

```suggestion
# function xyz does this and that
#
# $1: ... 
# $2: ...
# $3: ...
generate_cbor () {
```